Shared/Supporting Files/Extensions/Array+RawRepresentable.swift

```diff
@@ -15,10 +15,10 @@
 import Foundation
 
 /// An extension allowing an array to be used with the app storage property wrapper.
-extension Array: RawRepresentable where Element == String {
+extension Array where Element == String {
```

Comment:
This could be simplified:

```suggestion
extension Array<String> {
```